src/graphql/schema.graphql

```diff
@@ -222,13 +236,30 @@ input GamePieceCoordinatesInput {
 }
 
 input GamePieceRangedAttackActionInput {
-  targetGamePieceId: ID!
+  targetGamePieceId: Int!
```

Comment:
Note that we're changing this from `ID!` to `Int!`. This is because the IDs of all records will be integers, and this typing makes this a bit stricter and clearer. It also stops us from having to expect IDs to be `string | number` on the back end. I will plan to do this elsewhere as I address other areas.